packages/realm/src/index.ts

```diff
@@ -24,7 +24,7 @@ export { List } from "./List";
 export { Dictionary } from "./Dictionary";
 export { Set } from "./Set";
 export type { ObjectChangeSet, ObjectChangeCallback } from "./ObjectListeners";
-export { OrderedCollection, CollectionChangeSet, CollectionChangeCallback } from "./OrderedCollection";
```

Comment:
This is actually unrelated (?) but I left it because I am having issues building while this is in place, maybe worth looking into. Will remove from PR.

Reply:
I've already pushed a fix for this 👍